SharpPcap/LibPcap/NativeLibraryHelper.cs

```diff
@@ -35,7 +35,7 @@ public static void SetDllImportResolver(Assembly assembly, DllImportResolver res
                     null
                 );
 
-            setDllImportResolverMethod?.Invoke(null, new object[] {
+            setDllImportResolverMethod.Invoke(null, new object[] {
```

Comment:
`setDllImportResolverMethod` is `null` when trimming the application. I added the null check before to proceed, but this introduces other issues afterwards.